start-configuration

```diff
@@ -6,14 +6,7 @@ shopt -s nullglob
 export HOME=/data
 
 if [ ! -e /data/eula.txt ]; then
-  if [ "$EULA" != "" ]; then
-    echo "# Generated via Docker on $(date)" > eula.txt
-    echo "eula=$EULA" >> eula.txt
-    if [ $? != 0 ]; then
-      echo "ERROR: unable to write eula to /data. Please make sure attached directory is writable by uid=${UID}"
-      exit 2
-    fi
-  else
+  if [ "$EULA" != "TRUE" ]; then
```

Comment:
A bash trick I have learned since this original code is the case modification expansion `${parameter^^pattern}` ([described here](http://manpages.ubuntu.com/manpages/bionic/en/man1/bash.1.html#expansion)). So, this could be even more robust by changing this to:

```suggestion
  if [ "${EULA^^}" != "TRUE" ]; then
```